client/components/async-loader/test/index.js

```diff
@@ -66,3 +66,10 @@ test( 'creates wrapped components only on state change', async () => {
 		expect( renderCount ).toBe( 2 );
 	} );
 } );
+
+export const invalid = async () => {
+	1;
+};
+export const valid = async () => {
+	await Promise.resolve();
+};
```

Comment:
were these intentional or accidentally merged in?

Reply:
They were never intended to be added. I was investigating the `require-await` rule for https://github.com/Automattic/wp-calypso/pull/30697

Removal: https://github.com/Automattic/wp-calypso/pull/32540